python/ml_wrappers/model/image_model_wrapper.py

```diff
@@ -57,6 +57,7 @@
 BOXES = 'boxes'
 LABELS = 'labels'
 SCORES = 'scores'
+DEVICE = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
```

Comment:
It looks like this fails the minimal dependencies installed PR gate as torch is not available there.
One solution might be to do this within a try/catch?
A better solution might be to just put this logic inside the method _wrap_image_model, and use "auto" as the passed in string instead of the device.